index.d.ts

```diff
@@ -1131,7 +1131,7 @@ declare module 'mongoose' {
      * [methods](http://mongoosejs.com/docs/guide.html#methods).
      */
     // eslint-disable-next-line @typescript-eslint/ban-types
-    loadClass(model: Function): this;
+    loadClass(model: Function, onlyVirtuals: boolean): this;
```

Comment:
Actually, on second thought, `onlyVirtuals` should be optional, so this should be `onlyVirtuals?: boolean`.

`onlyVirtuals?: boolean` means you can call `loadClass()` without specifying the `onlyVirtuals` parameter.